apps/pm-user/src/components/organisms/sideBar/SideBar.js

```diff
@@ -1,28 +1,35 @@
 import React from 'react';
+import { DEFAULT_SELECTED_ITEM_KEY, MENU_ITEMS } from './constants';
 import { Menu } from '../../atoms';
-import { ProfileSummary } from "../../molecules";
+import { ProfileSummary } from '../../molecules';
+import { QuickInfoBar } from '../../molecules';
 import styles from './sideBar.module.scss';
-import { DEFAULT_SELECTED_ITEM_KEY, MENU_ITEMS } from './constants';
 
 function SideBar() {
-
   const handleMenuItemClick = ({ key }) => {
     //some code here
   };
 
-
   return (
     <div className={styles.container}>
-    <div className={styles.profileSummary}>
-        <ProfileSummary imageChangeLink="#" userDetails="Software Developer, 32" userImageSrc="https://zos.alipayobjects.com/rmsportal/jkjgkEfvpUPVyRjUImniVslZfWPnJuuZ.png" userName="Vinit Sharma" />
+      <div className={styles.profileSummary}>
+        <ProfileSummary
+          imageChangeLink="#"
+          userDetails="Software Developer, 32"
+          userImageSrc="https://zos.alipayobjects.com/rmsportal/jkjgkEfvpUPVyRjUImniVslZfWPnJuuZ.png"
+          userName="Vinit Sharma"
+        />
+      </div>
+      <div className={styles.quickInfoBarWraper}>
+        <QuickInfoBar />
       </div>
       <div className={styles.menu}>
         <Menu
           defaultSelectedKeys={[DEFAULT_SELECTED_ITEM_KEY]}
           items={MENU_ITEMS}
           onClick={handleMenuItemClick}
```

Comment:
is handleMenuItemClick required ?
If yes, then there is no code inside it... (lines 9-11).

So, is this onClick necessary ?

Reply:
it is Chaitnay's code, not mine. please ask him